docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -173,6 +173,7 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 
 NOTE: For restore to be successful you have to give restore configuration at the `Hazelcast` custom resource creation. Otherwise, triggering restore on currently running members will likely to fail. 
 
+NOTE: It is not allowed to override the existing restore data with a backup. This means that if there is a PersistentValue left from the previous Hazelcast cluster, you cannot restore a backup into this PersistentValue.
```

Comment:
Two notes one after another is not great; suggest making this a single note, as shown in the next comment

Reply:
I was actually planning to remove first note 👍
I will update it after we decide.